src/composables/multisig.ts

```diff
@@ -0,0 +1,218 @@
+        }
+        consensus.expiration_height = Number(consensus.expiration_height);
+        consensus.confirmations_required = Number(consensus.confirmations_required);
+
+        return {
+          ...camelcaseKeysDeep(consensus),
+          ...otherMultisig,
+          nonce: Number(nonce),
+          signers,
+          contractId,
+          balance: (await sdk.balance(otherMultisig.gaAccountId)) || 0,
+          address: otherMultisig.gaAccountId,
+        };
+      }),
+    ))
+      .sort((a, b) => b.balance - a.balance)
+      .sort((a, b) => b.confirmedBy.length - a.confirmedBy.length)
+      .sort((a) => a.confirmedBy
+        .some((c: string) => accounts.value.map(({ address }) => address).includes(c)) && -1);
+
+    if (!activeGaAccountId.value) {
+      setActiveGaAccountId(result[0]?.gaAccountId);
+    }
+
+    multisig.value = result;
+    storeMultisig(result);
+  }
+
+  async function buildSpendTx(
+    senderId: string,
+    recipientId: string,
+    amount: number,
+  ) {
+    const sdk = await getSdk();
+    return sdk.spendTx({
+      senderId,
+      recipientId,
+      amount,
+    });
+  }
+
+  async function postSpendTx(tx: string, txHash: string) {
+    return postJson(`${activeNetwork.value.multisigBackend}/tx`, { body: { hash: txHash, tx } });
+  }
+
+  async function proposeTx(spendTx: string, contractId: string) {
+    const sdk = await getSdk();
+    const expirationHeight = (
+      (await sdk.api.getTopHeader())?.height + MULTISIG_TRANSCATION_EXPARATION_HEIGHT
+    );
+    const spendTxHash = new Uint8Array(Crypto.hash(
+      // @ts-ignore
+      Buffer.concat([Buffer.from(sdk.getNetworkId()), decode(spendTx)]),
+    ));
+
+    const gaContractRpc = await sdk.getContractInstance({
+      source: multisigContract,
+      contractAddress: contractId,
+    });
+
+    await gaContractRpc.methods.propose.send(spendTxHash, {
+      FixedTTL: [expirationHeight],
+    });
+    return Buffer.from(spendTxHash).toString('hex');
+  }
+
+  async function confirmTx(contractId: string, spendTxHash: string) {
+    const sdk = await getSdk();
+    const expirationHeight = (
+      (await sdk.api.getTopHeader())?.height + MULTISIG_TRANSCATION_EXPARATION_HEIGHT
+    );
+    const gaContractRpc = await sdk.getContractInstance({
+      source: multisigContract,
+      contractAddress: contractId,
+    });
+
+    await gaContractRpc.methods.confirm.send(spendTxHash, {
+      FixedTTL: [expirationHeight],
+    });
+  }
+
+  async function revokeTx(spendTxHash: string, contractId: string) {
+    const sdk = await getSdk();
+    const gaContractRpc = await sdk.getContractInstance({
+      source: multisigContract,
+      contractAddress: contractId,
+    });
+
+    await gaContractRpc.methods.revoke.send(
+      Uint8Array.from(Buffer.from(spendTxHash, 'hex')),
+    );
+  }
+
+  async function sendTx(accountId: string, spendTx: string, nonce: number) {
+    const s = await Universal({
```

Comment:
I strongly suggest to avoid using one letter variable names.

Also please align with Subhod's changes on the `drySdk`: https://github.com/aeternity/superhero-wallet/pull/1785/files#diff-e6d7101251daf167a357f2c6276f092dc6b657ba30c3cbdd2082a144a124189d